### PR TITLE
DOC: in ssh forwarding, user git must be allowed to run docker

### PR DESCRIPTION
**Pull request by recumbentbirder**
Added to doc for rootless Docker installation:  for SSH passthrough, the ssh user (git) has to be able to run docker.

### CODE REVIEW

File: docs/content/installation/with-docker-rootless.en-us.md [hunk 1]
```diff
@@ -349,6 +349,7 @@ Match User git
   AuthorizedKeysCommandUser git
   AuthorizedKeysCommand /usr/bin/docker exec -i gitea /usr/local/bin/gitea keys -c /etc/gitea/app.ini -e git -u %u -t %t -k %k
 ```
+For this to work, the user `git` hast to be allowed to run `docker`.
```
Comment (techknowlogick) [inline docs/content/installation/with-docker-rootless.en-us.md]:
```suggestion

For this to work, the user `git` has to be allowed to run `docker`.
```

File: docs/content/installation/with-docker-rootless.en-us.md [hunk 2]
```diff
@@ -350,6 +350,8 @@ Match User git
   AuthorizedKeysCommand /usr/bin/docker exec -i gitea /usr/local/bin/gitea keys -c /etc/gitea/app.ini -e git -u %u -t %t -k %k
 ```
 
+For this to work, the user `git` has to be allowed to run `docker`.
```
Comment (techknowlogick) [inline docs/content/installation/with-docker-rootless.en-us.md]:
```suggestion
For this to work, the user `git` has to be allowed to run the `docker` cli command. Please read through the [security considerations](https://docs.docker.com/engine/security/#docker-daemon-attack-surface) of providing non-root linux users access to the docker daemon.
```

**Review by denyskon — CHANGES_REQUESTED** [review]
Please submit the PR to main branch, it will be backported automatically then.

**Comment by techknowlogick** [comment]
@denyskon for versioned docs, when clicking on edit it takes you to create a PR against the branch you were viewing.

**Comment by denyskon** [comment]
Maybe we should change that?

**Pull request merged**